### PR TITLE
test: clean up todos left in #3074, part of #3075

### PR DESCRIPTION
**Pull request by hyf0**

### CODE REVIEW

**Comment by hyf0** [comment]
* **#3080** <a href="https://app.graphite.dev/github/pr/rolldown/rolldown/3080?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#3079** <a href="https://app.graphite.dev/github/pr/rolldown/rolldown/3079?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/rolldown/rolldown/3079?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* **#3078** <a href="https://app.graphite.dev/github/pr/rolldown/rolldown/3078?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: crates/rolldown/tests/esbuild/default/external_with_wildcard/_config.json [hunk 1]
```diff
@@ -5,7 +5,13 @@
...
+    ],
+    "external": [
+      "/assets/images/test.jpg",
+      "./file.png",
+      "/dir/x/file.gif",
+      "/dir//file.gif"
     ]
   },
-  "expectError": true // TODO(hyf0): Only for pass tests in #3074. Should investigate if it should raise an error.
+  "expectError": true
```
Comment (hyf0) [inline crates/rolldown/tests/esbuild/default/external_with_wildcard/_config.json:16]: expect error: https://github.com/evanw/esbuild/blob/a08f30db4a475472aa09cd89e2279a822266f6c7/internal/bundler_tests/bundler_default_test.go#L2571-L2601

File: crates/rolldown/tests/esbuild/ts/ts_implicit_extensions_missing/_config.json [hunk 2]
```diff
@@ -7,5 +7,5 @@
       }
     ]
   },
-  "expectError": true // TODOO(hyf0): Only for pass tests in #3074. Should investigate if it should raise an error.
+  "expectError": true
```
Comment (hyf0) [inline crates/rolldown/tests/esbuild/ts/ts_implicit_extensions_missing/_config.json:10]: https://github.com/evanw/esbuild/blob/a08f30db4a475472aa09cd89e2279a822266f6c7/internal/bundler_tests/bundler_ts_test.go#L1458-L1482

File: crates/rolldown/tests/esbuild/ts/ts_import_missing_file/_config.json [hunk 3]
```diff
@@ -7,5 +7,5 @@
       }
     ]
   },
-  "expectError": true // TODO(hyf0): Only for pass tests in #3074. Should investigate if it should raise an error.
+  "expectError": true
```
Comment (hyf0) [inline crates/rolldown/tests/esbuild/ts/ts_import_missing_file/_config.json:10]: https://github.com/evanw/esbuild/blob/a08f30db4a475472aa09cd89e2279a822266f6c7/internal/bundler_tests/bundler_ts_test.go#L345-L361

**Review by underfin — APPROVED** [review]

**Pull request merged**